Codecademy/gamut
# docs: Add back old UX Writing docs into SB8

### Overview


Add back in UX Writing section to Storybook with some minor updates


### PR Checklist

- [x] Related to JIRA ticket: [GM-960]

#### Testing Instructions



1. Go to the preview link and scroll down to the "UX Writing" section.
2. Navigate through the pages to see that they load correctly
3. (Optionally) click on links to check they work and link to their appropriate page
4. (Optionally) proofread along the way
5. ...
6. Profit! 




[GM-960]: https://skillsoftdev.atlassian.net/browse/GM-960?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
